### ISSUE TITLE
Learning Pod 1 Major Project: Tik Tok

### ISSUE THREAD

**Issue by emilylehoux** [cmadland/edci338-202201#81]
Hey! Here is our issue for the project. Make sure you can access this easily. Thanks.

**Comment by cmadland** [comment]
@emilylehoux do you see the 'Projects' section on the right side of this page? If so, is the 'EDCI338 Major Project' project an option?

**Comment by emilylehoux** [comment]
Yeah, I tried adding it, but nothing popped up. There was no option to click to add.

Hope this helps,

Emily
________________________________

**Comment by cmadland** [comment]
Ok, I've added you to the project...can you see it now?

**Comment by emilylehoux** [comment]
- [ ] Personas  
- [ ] Demographic Information  
- [ ] Digital Identity  
- [ ] Privacy and Surveillance  
- [ ] Access and Inclusion  
- [ ] Trust and Misinformation  
- [ ] Public Discourse  
- [ ] Education  
- [ ] Media Literacy  
- [ ] Reflections

**Comment by cmadland** [comment]
Hey everyone...came across this podcast today.

https://anchor.fm/cheyandpav/episodes/Using-TikTok-in-the-Classroom---106-e1elf9o/a-a7erk5o 
@emilylehoux @miaharmitage @EDCI338 @tiannaroutley

**Comment by cmadland** [comment]
Hey!
I've been through each of your Showcase posts and just wanted to point out that it is obvious to me that you four are really engaging with the topics of the course in your conversations.

i appreciate your work!

cm

**Comment by miaharmitage** [comment]
@cmadland Thank you for sharing the podcast link! I will for sure be giving it a listen tomorrow. It is super cool that the podcast is from Canadian teachers, I'm really looking forward to hearing their perspectives and hopefully, integrating this into our major project.
Also, thank you for the kind comment about my learning pods showcase posts. We all have really enjoyed the content and conversations we've been able to have this far!

**Comment by cmadland** [comment]
Article for you... https://apple.news/AmvR48Rb0QgmEbysZ8Zb2uA
